README.md

```diff
@@ -98,6 +98,82 @@ The following code example shows the three main steps to use Alibaba Cloud Java
      }
  }
 ```
+
+### Use the default credential provider chain
+
+The default credential provider chain looks for available credentials, with following order:
+
+1.System Properties
+
+Look for environment credentials in system properties. If the `alibabacloud.accessKeyId` and `alibabacloud.accessKeyIdSecret` system properties are defined and not empty, the program will use them to create default credentials.
+
+2.Environment Credentials
+
+Look for environment credentials in environment variable. If the `ALIBABA_CLOUD_ACCESS_KEY_ID` and `ALIBABA_CLOUD_ACCESS_KEY_SECRET` environment variables are defined and are not empty, the program will use them to create default credentials.
+
+3.Credentials File
+
+If there is `~/.alibabacloud/credentials default file (Windows shows C:\Users\USER_NAME\.alibabacloud\credentials)`, the program automatically creates credentials with the specified type and name. The default file may not exist, but a parse error throws an exception. Configuration name lowercase.This configuration file can be shared between different projects and between different tools. Because it is outside the project and will not be accidentally committed to the version control. The path to the default file can be modified by defining the `ALIBABA_CLOUD_CREDENTIALS_FILE` environment variable. Use the configuration via `AuthUtils.setClientType("default")`.

```

Comment:
The default file may not exist, but a parse error throws an exception ->
The default file is not necessarily exist, but a parse error will throw an exception